example/llrt-sam/template.yaml

```diff
@@ -0,0 +1,55 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  llrt-sam
+
+  Sample SAM Template for llrt-sam
+
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
+Globals:
+  Function:
+    Timeout: 3
+
+Resources:
+
+  LlrtLayer:
+    Type: AWS::Serverless::LayerVersion
+    Properties:
+      ContentUri: ../../llrt-lambda-arm64.zip
+      CompatibleRuntimes:
+        - provided.al2023
+      CompatibleArchitectures:
+        - arm64
+  HelloWorldFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: hello-world/
+      Handler: app.lambdaHandler
+      Runtime: provided.al2023
+      Architectures:
+        - arm64
+      Layers:
+        - !Ref LlrtLayer
+    Metadata: # Manage esbuild properties
+      BuildMethod: esbuild
+      BuildProperties:
+        External:
+          - '@aws-sdk'
+          - '@smithy'
+          - uuid
```

Comment:
Does this mean these are already present in the layer / runtime?

Reply:
Yes! See https://github.com/awslabs/llrt?tab=readme-ov-file#using-aws-sdk-v3-with-llrt